test/functional/rpc_mempool_entry_fee_fields_deprecation.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test deprecation of fee fields from top level mempool entry object"""
+
+from test_framework.blocktools import COIN
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal
+from test_framework.wallet import MiniWallet
+
+
+def assertions_helper(new_object, deprecated_object, deprecated_fields):
+    for field in deprecated_fields:
+        assert field in deprecated_object
+        assert field not in new_object
+
+
+class MempoolFeeFieldsDeprecationTest(BitcoinTestFramework):
+    def set_test_params(self):
+        self.num_nodes = 2
+        self.extra_args = [[], ["-deprecatedrpc=fees"]]
+
+    def run_test(self):
+        # we get spendable outputs from the premined chain starting
+        # at block 76. see BitcoinTestFramework._initialize_chain() for details
+        self.wallet = MiniWallet(self.nodes[0])
+        self.wallet.rescan_utxos()
+
+        # we create the tx on the first node and wait until it syncs to node_deprecated
+        # thus, any differences must be coming from getmempoolentry or getrawmempool
+        txid = self.wallet.send_self_transfer(from_node=self.nodes[0])["txid"]
+        self.sync_mempools(nodes=self.nodes)
```

Comment:
This sync can take several seconds since bitcoind will add some random delay on transaction relay to improve privacy. Since testing tx relay isn't the point of this test, you could make the test run reliably faster by submitting the transaction directly to node 1 with the `sendrawtransaction` RPC.

Reply:
thanks for the tip! i did notice the new test was taking longer than i expected. fixed!

Reply:
change since your last ACK: `git range-diff  7330beb...2f9515f`